charts/airbyte/templates/pod-sweeper/deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
       {{- end }}
       containers:
       - name: airbyte-pod-sweeper
-        image: "{{ .Values.podSweeper.image.repository }}:{{ .Values.podSweeper.image.tag}}"
+        image: {{ include "airbyte.podSweeperImage" . }}
```

Comment:
why change this to use `include` instead?

Reply:
ah is this using the custom functions defined in `_helpers.tpl`?

Reply:
Yeah. I'm using a helper function from the bitnami common library chart to make it easier to support `global.imageRegistry`